test/include/RAJA_test-index-types.hpp

```diff
@@ -56,9 +56,9 @@ using SignedIdxTypeList = camp::list<RAJA::Index_type,
 //
 using StrongIdxTypeList = camp::list<RAJA::Index_type,
                                      int,
-                                     StrongIndexType,
+                                     //StrongIndexType,
```

Comment:
Why are the "Strong" index types turned excluded now?

Reply:
This was turned off due to how we are checking the lambda arguments against the expected args. Until now we just assumed Index_type for the first argument in a lambda and that works fine except when we are trying to use a strong type. I'm working on a fix for this now.

Reply:
Fixed.

Reply:
Excellent. Good work!